web/views/page_view.ex

```diff
@@ -59,5 +59,8 @@ defmodule Askbywho.PageView do
     |> raw
   end
 
-
+  def on_qa do
+    IO.puts("environment var is_qa is #{inspect System.get_env("is_qa")}")
+    System.get_env("is_qa") == "yes"
```

Comment:
what do you mean "qa thinks it is prod"?  Is QA an actual different server?  If its a diff server, you should use diff settings than prod, so having its own config/qa.env file and then you can follow normal convention for types of environments